### ISSUE TITLE
EMAIL Config Documentation Error

### ISSUE THREAD

**Issue by ezrajrice** [tethysplatform/tethys#557]
The given example for the email configurations is incorrect for portal_config.yml.
Current Instructions:
```yaml
settings:
  ...
  EMAIL_CONFIG:
    EMAIL_BACKEND: 'django.core.mail.backends.smtp.EmailBackend'
    EMAIL_HOST: 'localhost'
    EMAIL_PORT: 25
    EMAIL_HOST_USER: ''
    EMAIL_HOST_PASSWORD: ''
    EMAIL_USE_TLS: False
    DEFAULT_FROM_EMAIL: 'EXAMPLE <noreply@example.com>'
```
These don't get read by the settings.py file due to the result not matching what Django is expecting. Because of this, all emails sent with this configuration will use the Django default settings and you will see all emails sent from a user such as "webmaster@localhost". These settings should not be nested inside the EMAIL_CONFIG setting. It should read:
```yaml
settings:
  ...
  # EMAIL_CONFIG:
  EMAIL_BACKEND: 'django.core.mail.backends.smtp.EmailBackend'
  EMAIL_HOST: 'localhost'
  EMAIL_PORT: 25
  EMAIL_HOST_USER: ''
  EMAIL_HOST_PASSWORD: ''
  EMAIL_USE_TLS: False
  DEFAULT_FROM_EMAIL: 'EXAMPLE <noreply@example.com>'
```

**Comment by swainn** [comment]
@sdc50 this is another instance where the documentation for the settings needs to be updated.

**Comment by swainn** [comment]
@ezrajrice we just merged PR #573 this weekend that should address this issue. Both the nested approach and the non-nested approach should work. See updated documentation here: http://docs.tethysplatform.org/en/latest/tethys_portal/configuration.html
